apiserver/facades/agent/uniter/storage.go

```diff
@@ -449,3 +461,22 @@ func watchStorageAttachment(
 	watchers = append(watchers, st.WatchStorageAttachment(storageTag, unitTag))
 	return common.NewMultiNotifyWatcher(watchers...), nil
 }
+
+// watcherAdaptor adapts a core watcher to a state watcher.
+// TODO(wallyworld) - this will be obsolete once storage is implemented in dqlite.
```

Comment:
I had assumed we fixed all these, but I guess that's not true.

Reply:
Yeah, there's 2 - here and apiserver common cloudspec